test/controllers/event_controller_test.exs

```diff
@@ -12,7 +12,8 @@ defmodule Dwylbot.EventTestController do
     %{payload: "pr_no_description", event: "pull_request" },
     %{payload: "pr_merge_conflict", event: "pull_request" },
     %{payload: "pr_no_assignee", event: "pull_request" },
-    %{payload: "request_reviewer", event: "pull_request" }
+    %{payload: "request_reviewer", event: "pull_request" },
+    %{payload: "failing_test", event: "status" },
```

Comment:
Nice, adding this trailing comma will make future diffs easier slightly to see 👍